app/waf/src/controllers/application.controller.ts

```diff
@@ -0,0 +1,138 @@
+import {
+  Count,
+  CountSchema,
+  Filter,
+  repository,
+  Where,
+} from '@loopback/repository';
+import {
+  post,
+  param,
+  get,
+  getFilterSchemaFor,
+  getWhereSchemaFor,
+  patch,
+  put,
+  del,
+  requestBody,
+} from '@loopback/rest';
+import {Application} from '../models';
+import {ApplicationRepository} from '../repositories';
+
+export class AdcController {
+  constructor(
+    @repository(ApplicationRepository)
+    public applicationRepository: ApplicationRepository,
+  ) {}
+
+  @post('/Applications', {
```

Comment:
Suggest not to use upper case letter in url path.

Reply:
typo. fixed.